### PR TITLE
S1 t3 basic html css structure

### PR DESCRIPTION
**Pull request by robin-hodge**
First pull request from the third task onto the story branch

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -0,0 +1,51 @@
+<?php
+?>
+
+<!DOCTYPE HTML>
+<HTML>
+    <head>
+        <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css">
+        <link rel="stylesheet" type="text/css" href="styles/styles.css" >
```
Comment (amyas-gilbert) [inline index.php]: left a space before closing the link tag

Comment (robin-hodge) [reply index.php]: Changed!

Comment (robin-hodge) [reply index.php]: Changed!

File: index.php [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        <h3 id="dream title">Walls in house made of sponge cake!</h3>
+        <h4 id="dream or nightmare">Dream/Nightmare</h4>
+        <h4>01/01/1970</h4>
+        <p id="dream description">I had a dream where I saw cracks appearing in the walls of my house, 
+            that started to grow and grow. As they grew, they started to look like sponge cake,
+             and fell away to reveal huge holes in the wall. How odd!! I spoke with my therapist about this,
+             and she said it might be a metaphore about all of the things that I am learning, and feeling like 
+             my brain almost doesn't have space to store it all! I'm not toally sure I agree with this, but who 
+             knows, she is the pro....
+        </p>
```
Comment (amyas-gilbert) [inline index.php]: not sure, but I think all the whitespace in the <p> tags, which I assume you put in for the indentation in the code,  could lead to unforeseen problems down the road?

File: index.php [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+<HTML>
+    <head>
+        <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css">
+        <link rel="stylesheet" type="text/css" href="styles/styles.css" >
+    </head>
+    <body>
+        <h1 id="page title">Your Dream Diary</h1>
+        <h2 id="dreams title">Dreams you have had</h2>
+        <h3 id="dream title">Walls in house made of sponge cake!</h3>
+        <h4 id="dream or nightmare">Dream/Nightmare</h4>
```
Comment (amyas-gilbert) [inline index.php]: just wondering why you added id to all these tags. if it's for styling i feel like you could just refer to the <h> and <p> tags directly as they each seem to always apply to the same things

Comment (amyas-gilbert) [reply index.php]: sorry: 
just wondering why you added id to all these tags. if it's for styling i feel like you could just refer to the h and
p tags directly as they each seem to always apply to the same things

File: index.php (hunk 2 above)
Comment (amyas-gilbert) [reply index.php]: sorry:
not sure, but I think all the whitespace in the p
tags, which I assume you put in for the indentation in the code, could lead to unforeseen problems down the road?

**Review by amyas-gilbert — CHANGES_REQUESTED** [review]
few potential changes...

File: index.php (hunk 2 above)
Comment (robin-hodge) [reply index.php]: done!

File: index.php (hunk 3 above)
Comment (robin-hodge) [reply index.php]: I need to check to see if this is correct for screen readers etc. If it is, I will leave, otherwise I will change attribute type

**Review by amyas-gilbert — APPROVED** [review]
aaaalll good!

**Review by amyas-gilbert — APPROVED** [review]
go for it

**Review by amyas-gilbert — APPROVED** [review]
!!

**Pull request merged**